src/StashUserscriptLibrary.js

```diff
@@ -1,5 +1,5 @@
 // Stash Userscript Library
-// Exports utility functions and a Stash class that emits events whenever a GQL response is received and whenenever a page navigation change is detected
+// Exports utility functions and a Stash class that emits events whenever a GQL response is received and whenever a page navigation change is detected
 // version 0.33.0
```

Comment:
I would increase the version.

Reply:
I would think this is a minor revision so 0.33.1?